app/src/main/java/com/framgia/soundclound/screen/detailgenre/GenreDetailActivity.java

```diff
@@ -24,4 +26,10 @@ protected void onCreate(Bundle savedInstanceState) {
                 .getString(Constant.EXTRA_GENRE));
         activityGenreDetailBinding.setViewModel(mViewModel);
     }
+
+    public static Intent getInstance(Context context, String genre) {
```

Comment:
Hàm này nên đặt trước onCreate
2.8 Class member ordering
This is not the only correct solution, but it is recommended to use the following order :

Constants
Fields
Constructors
Override methods and callbacks (public or private)
Public methods
Private methods
Inner classes or interfaces
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md